menoh/menoh_chainer_compiler.cpp

```diff
@@ -749,6 +759,9 @@ menoh_error_code menoh_model_run(menoh_model_handle model) {
                 auto const& array = chainerx::AsContiguous(output.second->GetArray());
                 auto const& shape = array.shape();
                 auto bytesize = shape.GetTotalSize() * chainerx::GetItemSize(array.dtype());
+                assert(model->variable_profiles.find(output.first) != model->variable_profiles.end());
```

Comment:
CHECK_NE please

Reply:
OK. I will modify.

Reply:
I can't use CHECK_NE here because iterator is not printable.
I used `CHECK(a != b)` instead of `CHECK_NE(a, b)`